doc/basic_configuration.rst

```diff
@@ -35,16 +42,15 @@ The following options must be set in *all* build files::
 
     notifications:
       committers: false
-    maintainers
-      committers: false
+      maintainers: false
 
 
 Update administrator notification
 ---------------------------------
 
 Change the email address which gets notified about any administrative jobs.
 
-In the entry point yaml::
+In `ros-infrastructure/ros_buildfarm_config <https://github.com/ros-infrastructure/ros_buildfarm_config>`_'s `index.yaml`::
```

Comment:
I don't think this should reference a specific repository since that is not the same for any custom buildfarm. Also the referenced branch in this case is incorrect.

Mentioning the filename `index.yaml` explicitly would indeed be good (rather than calling it "entry point yaml").

Reply:
Thanks! Good point. I'll remove the link and make it more clear.